frontend/server/src/Controllers/Submission.php

```diff
@@ -129,4 +129,118 @@ public static function getLatestUserSubmissionsForTypeScript(\OmegaUp\Request $r
+            )
+        );
+        if (is_null($courseSubmissionInfo)) {
+            throw new \OmegaUp\Exceptions\NotFoundException(
+                'courseSubmissionNotFound'
+            );
+        }
+
+        $feedback = $r->ensureString(
+            'feedback',
+            fn (string $feedback) => \OmegaUp\Validators::stringOfLengthInRange(
+                $feedback,
+                1,
+                200
+            )
+        );
+
+        $course = \OmegaUp\DAO\Courses::getByAlias(
+            $courseSubmissionInfo['course_alias']
+        );
+        if (is_null($course)) {
+            throw new \OmegaUp\Exceptions\NotFoundException(
+                'courseNotFound'
+            );
+        }
+
+        if (
+            !\OmegaUp\Authorization::isCourseAdmin(
+                $r->identity,
+                $course
+            )
+        ) {
+            throw new \OmegaUp\Exceptions\ForbiddenAccessException();
+        }
+
+        // Verificar si ya existe un feedback para esto.
+        try {
+            \OmegaUp\DAO\DAO::transBegin();
+
+            \OmegaUp\DAO\SubmissionFeedback::create(
+                new \OmegaUp\DAO\VO\SubmissionFeedback([
+                    'identity_id' => $r->identity->identity_id,
+                    'submission_id' => $submission->submission_id,
+                    'feedback' => $feedback,
+                ])
+            );
+
+            if (!is_null($courseSubmissionInfo['author_id'])) {
+                \OmegaUp\DAO\Notifications::create(
+                    new \OmegaUp\DAO\VO\Notifications([
+                        'user_id' => $courseSubmissionInfo['author_id'],
+                        'contents' =>  json_encode([
+                            'type' => \OmegaUp\DAO\Notifications::COURSE_SUBMISSION_FEEDBACK,
+                            'body' => [
+                                'localizationString' => new \OmegaUp\TranslationString(
+                                    'notificationCourseSubmissionFeedback'
+                                ),
+                                'localizationParams' => [
+                                    'problemAlias' => $courseSubmissionInfo['problem_alias'],
+                                    'courseName' => $course->name,
+                                ],
+                                'url' => "/course/{$course->alias}/assignment/{$courseSubmissionInfo['assignment_alias']}/#problems/{$courseSubmissionInfo['problem_alias']}/",
+                                'iconUrl' => '/media/info.png',
+                            ]
+                        ]),
+                    ])
+                );
+            }
+        } catch (\Exception $e) {
+            \OmegaUp\DAO\DAO::transRollback();
+
+            if (\OmegaUp\DAO\DAO::isDuplicateEntryException($e)) {
+                throw new \OmegaUp\Exceptions\DuplicatedEntryInDatabaseException(
+                    'submissionFeedbackAlreadyExists'
+                );
+            }
```

Comment:
```suggestion
            }
            throw $e;
```

si no, cualquier otro tipo de falla va a decir que sí se pudo hacer la operación!